Symfony/CS/Tokenizer/Token.php

```diff
@@ -99,9 +100,22 @@ public function equals($other)
         $selfPrototype = $this->getPrototype();
 
         foreach ($otherPrototype as $key => $val) {
-            if ($selfPrototype[$key] !== $val) {
+            // Make sure the token has such key
```

Comment:
OK, to point it out:
[Follow the conventions used in the project. (from CONTRIBUTING.md)](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.5/CONTRIBUTING.md)

And since project's standards are Symfony Coding Standards (again CONTRIBUTING.md), then...
[golden rule: Imitate the existing Symfony code](http://symfony.com/doc/current/contributing/code/standards.html)